crates/router/src/connector/payme/transformers.rs

```diff
@@ -507,6 +587,100 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for PayRequest {
         }
     }
 }
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct PaymePayloadData {
```

Comment:
Rename this to PaymeRedirectResponseData. Take this in another PR

Reply:
Sure👍